nmostesting/NMOSTesting.py

```diff
@@ -1139,6 +1139,13 @@ def main(args):
     print(" * Testing tool running on 'http://{}:{}'. Version '{}'"
           .format(get_default_ip(), core_app.config['PORT'], TOOL_VERSION))
 
+    # Give an API or client that is already running a chance to use the mock services
+    # before running any test cases
+    if CONFIG.MOCK_SERVICES_WARM_UP_DELAY:

```

Comment:
Presumably we only need this delay when running non-interactive tests?  When running interactively I'm guessing this delay is neither necessary or desirable and the message wouldn't make sense. 

If that's the case then either the change I've suggested, or move the code block to the `else` of the `if` block in line 1150? Perhaps?
```suggestion
    if CONFIG.MOCK_SERVICES_WARM_UP_DELAY and "suite" in vars(CMD_ARGS):
```

Reply:
Hmm... I think theoretically you do need it in the interactive tests, say if you typed really fast/used browser-cached values?

Reply:
Fair point.

I still feel the message might be confusing for those operating in interactive mode - i.e. stating that testing will begin when they haven't yet initiated the tests.

Perhaps the message should be consistent with other wait message in the code base:
`"* Waiting {}s to allow discovery of mock resources"`